themes/edx.org/lms/static/sass/partials/base/_certificates.scss

```diff
@@ -0,0 +1,27 @@
+// Certificate overrides for edx.org
+
+.certificate-card {
+    &.mode-honor, &.mode-audit {
```

Comment:
Same comments as in the edge file.
Curious, there's no way to avoid duplication between these files in different themes where appropriate?

Reply:
I don't think there's a clean way to share Sass styles between themes, due to the way that the theme load path works. I wondered briefly about how to get it to be shared but decided against spending time on it.